### DISCUSSION TITLE
Mountain Duck: Add file synchronization filters

### DISCUSSION THREAD

**Discussion by galloppinggryphon** [iterate-ch/cyberduck#14748] [Ideas]
**Is your feature request related to a problem? Please describe.**
Cyberduck enables upload filtering of files/folders. E.g. preventing node_modules folders from being stored in the cloud. I cannot see that Mountain Duck supports this. Please correct me if I'm wrong...

I regret that this functionality is **THE** reason for wanting an alternative to the Onedrive/Google Drive sync clients. When Mountain Duck supports filtering, I'll consider getting a license.

**Describe the solution you'd like**
I like the idea of having a `.gitignore` file for files and folders that should not be uploaded. The idea comes from ticket #13753. 

Alternatively, Cyberduck's filtering option dialogue should be implemented for Mountain Duck, though ideally on a _per bookmark_ basis.

Thank you.

**Comment by dkocher** [comment]
I have added this to our backlog of feature requests in our internal issue tracker for _Mountain Duck_.

**Comment by Jaxenormus** [comment]
@dkocher any eta on this?

**Reply by dkocher** [reply]
This will be shipped in a 5.x release.

**Comment by tristanryerparke** [comment]
Has there been any progress on this feature?
